### PR TITLE
map themes

### PR DESCRIPTION
**Pull request by smontana30**
user can pick what map style they want

### CODE REVIEW

**Review by billyporter — APPROVED** [review]

**Review by Sarahhud — APPROVED** [review]

**Review by NicholasCobbFromGoogle — APPROVED** [review]

File: capstone-app/src/app/modules/maps/map/map.component.html [hunk 1]
```diff
@@ -15,6 +15,15 @@
     </button>
   </div>
 
+  <mat-form-field appearance="fill">
+    <mat-label >Map Theme</mat-label>
```
Comment (mbehrlich) [inline capstone-app/src/app/modules/maps/map/map.component.html]: Nit: Remove extraneous space in mat-label

File: capstone-app/src/app/modules/maps/map/map.component.ts [hunk 2]
```diff
@@ -46,6 +46,965 @@ export class MapComponent implements OnInit, OnChanges {
     'Meal Takeaway', 'Mosque', 'Movie Theater', 'Museum','Night Club', 'Park', 'Police', 
     'Restaurant', 'Rv Park', 'Shopping Mall', 'Stadium', 'Store', 'Subway Station', 'Supermarket',
     'Synagogue', 'Taxi Stand', 'Tourist Attraction', 'Train Station', 'Transit Station', 'University', 'Zoo'];
+  darkMode: google.maps.MapTypeStyle[] = [
```
Comment (mbehrlich) [inline capstone-app/src/app/modules/maps/map/map.component.ts:50]: I recommend exporting these from a separate style so as to not clutter the component file.

File: capstone-app/src/app/modules/maps/map/map.component.ts [hunk 3]
```diff
@@ -73,6 +1032,45 @@ export class MapComponent implements OnInit, OnChanges {
     }
   }
 
+  setMapStyle(mapType: string) {
+    if (mapType.includes('Retro')) {
```
Comment (mbehrlich) [inline capstone-app/src/app/modules/maps/map/map.component.ts]: Shouldn't this be '===' instead of 'includes'?

Comment (mbehrlich) [reply capstone-app/src/app/modules/maps/map/map.component.ts]: Below as well

File: capstone-app/src/app/modules/maps/map/map.component.ts [hunk 4]
```diff
@@ -73,6 +1032,45 @@ export class MapComponent implements OnInit, OnChanges {
     }
   }
 
+  setMapStyle(mapType: string) {
```
Comment (mbehrlich) [inline capstone-app/src/app/modules/maps/map/map.component.ts:1036]: optional: this.options really only needs to be set once.
```
let newMap: google.maps.MapOptions;
if (...) {
  newMap = {...};
} else if (...) {...}
this.options = newMap;
```

**Review by mbehrlich — APPROVED** [review]

**Pull request merged**